src/data/lumine-text.asm

```diff
@@ -0,0 +1,8 @@
+.loadtable "data/lumine-table.tbl"
+
+.strn   "\blank"
+.strn   "I'm \cness....\nIt's been a long road getting here...\n"
```

Comment:
Counterintuitive but for some reason the game never actually prints the first period after Ness's name. Just `I'm Ness...  It's` etc

Reply:
You sure about this one?

Reply:
https://www.youtube.com/watch?v=cFsibAm0Iy8